src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -2641,6 +2646,10 @@ public static final void updateCombatData(
             }
             break;
 
+          case VOID:
+            Preferences.increment("_voidFights");
```

Comment:
Can you max this out at 5 (there is an overload for doing so)? And can you add a check for the "time seems to stop" message, the failure of which should set this straight to 5.

Reply:
aye aye, cap'n